src/pexprs-allowsSkippingPrecedingSpace.js

```diff
@@ -22,6 +22,7 @@ pexprs.PExpr.prototype.allowsSkippingPrecedingSpace = common.abstract(
   Generally, these are all first-order expressions and (with the exception of Apply)
   directly read from the input stream.
 */
+pexprs.Syn.prototype.allowsSkippingPrecedingSpace =
```

Comment:
I would think that this should be down below, under "higher-order expressions that don't directly consume input" (where you put `Lex` as well). But, TBH I'm not why Lex wasn't already there.